bundle/src/main/java/com/adobe/acs/commons/adobeio/core/healthcheck/AdobeIOHealthcheck.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2013 - 2018 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.adobe.acs.commons.adobeio.core.healthcheck;
+
+import org.apache.sling.hc.api.HealthCheck;
+import org.apache.sling.hc.api.Result;
+import org.apache.sling.hc.util.FormattingResultLog;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+import com.adobe.acs.commons.adobeio.core.service.IntegrationService;
+
+@SuppressWarnings("WeakerAccess")
+@Component(service = HealthCheck.class,
+        property = {HealthCheck.NAME + "=ACS AEM Commons - Adobe I/O configuration",
+                HealthCheck.TAGS + "=adobeio",
+                HealthCheck.MBEAN_NAME + "=ACS AEM Commons - Adobe I/O health check"})
+public class AdobeIOHealthcheck implements HealthCheck {
+
+    @Reference
+    private IntegrationService jwtService;
```

Comment:
It looks to me like this integration only supports a single IntegrationService (and thus API Key) per instance. Is this an acceptable limitation?